http/middleware/response_header.go

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2021 by PACE Telematics GmbH. All rights reserved.
+// Created at 2021/05/10 by Alessandro Miceli
+
+package middleware
+
+import (
+	"context"
+	"net/http"
+	"strings"
+	"sync"
+
+	"github.com/pace/bricks/http/oauth2"
+	"github.com/pace/bricks/maintenance/log"
+)
+
+// ClientIDHeaderName name of the HTTP header that is used for reporting
+const ClientIDHeaderName = "Client-ID"
```

Comment:
Please check if this header is used otherwhere, i.e., whether this is a standard header that we might conflict with.